### ISSUE TITLE
Loss function: possible negaitve losses? 

### ISSUE THREAD

**Issue by EryiXie** [JunjH/Revisiting_Single_Depth_Estimation#30]
Thanks for your excellent work. However there are some codes about loss function make me quiet confused, for example:

https://github.com/JunjH/Revisiting_Single_Depth_Estimation/blob/a486b9a3fc708a4f320881ee3b31f94022e1c184/train.py#L110

If x<0.5, then torch.log(x + 0.5) < 0. So it means a negative loss value right?
